en/Contributing to Obsidian/Style guide.md

```diff
@@ -100,3 +100,53 @@ When referring to a character on the keyboard by name, add the character between
 
 - Add a hyphen in front of the word.
 - Add a `-` in front of the word.
+
+## Markdown
+
+Use newlines between Markdown blocks:
+
+**Recommended:**
+
+```md
+# Heading 1
+
+This is a section.
+
+1. First item
+2. Second item
+3. Third item
+```
+
+**Not recommended**
+
+```md
+# Heading 1
+This is a section.
+1. First item
+2. Second item
+3. Third item
+```
+
+Use lazy numbering for lists with more than three items:
+
+**Recommended**
```

Comment:
So, one downside to lazy numbering is that you can't make edits to the docs via Obsidian.

Reply:
I was about to say that I wish Obsidian automatically readjusted the numbers, but now checking again, it seems like it does! Could've sworn it didn't work the last time I looked. I do all my docs work inside Obsidian so I'm fine removing the lazy numbering style.